### PR TITLE
Enable Mobile Toggle of Navigation

### LINKED ISSUE #145: Make mobile nav functional

### PR DESCRIPTION
**Pull request by funkybunch**
Resolves #145 where the mobile nav was not toggling.

### CODE REVIEW

**Review by Dafondo — APPROVED** [review]
👍

File: src/components/Header.js [hunk 1]
```diff
@@ -9,6 +9,21 @@ import {
 import { DEFAULT_ROUTE, DATA_DICTIONARY_ROUTE, ABOUT_ROUTE } from '../constants'
 
 const Header = () => {
+  const [expanded, setExpanded] = useState(false)
+  const onClick = (): void => setExpanded((prvExpanded) => !prvExpanded)
```
Comment (mcelestini) [inline src/components/Header.js:13]: can we wrap this in `useCallback`?

File: src/components/Header.js [hunk 2]
```diff
@@ -9,6 +9,21 @@ import {
...
+
+  const menuItems = [
```
Comment (mcelestini) [inline src/components/Header.js:15]: and wrap this in `useMemo`?

File: src/components/Header.js (hunk 1 above)
Comment (mcelestini) [reply src/components/Header.js:13]: also, would you be open to naming this something more descriptive? We may have more than one onClick handler in the future. How about something like `handleMenuClick` or `onClickMenu`?

**Comment by mcelestini** [comment]
Doof... just realized that this is already merged. Looks good 👍

**Comment by funkybunch** [comment]
@mcelestini Since this is the nav component and used on every render I didn't think they were necessary.  But, if you think we should add those optimizations please feel free to create an issue so we don't forget and then we can put them in post-demo.

**Comment by mcelestini** [comment]
Maybe we should do a general front end optimization before we "release" and look for stuff like this. I'll create an issue 😄

**Pull request merged**